src/Data/Array/Accelerate/Language.hs

```diff
@@ -117,8 +117,8 @@ import Data.Array.Accelerate.Classes.Fractional
 import Data.Array.Accelerate.Classes.Integral
 import Data.Array.Accelerate.Classes.Num
 import Data.Array.Accelerate.Classes.Ord
-
 import Prelude                                                      ( ($), (.), Maybe(..), Char )
+import Data.Array.Accelerate.Sugar.Stencil
```

Comment:
The imports were structured before -- whether that's useful is debatable. But I think `import Prelude` should be separate because it should catch the eye.